linux/amzn/2014.09/foxpass_setup.py

```diff
@@ -145,18 +146,18 @@ def augment_sshd_config():
             w.write("AuthorizedKeysCommandUser\troot\n")
 
 
-# give "wheel" and "foxpass-sudo" groups sudo permissions without password
-def fix_sudo():
+# give "sudo" and chosen sudoers groups sudo permissions without password
+def fix_sudo(sudoers):
     os.system("sed -i 's/^# %wheel\tALL=(ALL)\tNOPASSWD: ALL/%wheel\tALL=(ALL)\tNOPASSWD:ALL/' /etc/sudoers")
     if not file_contains('/etc/sudoers', '^#includedir'):
         with open('/etc/sudoers', 'a') as w:
             w.write('\n#includedir /etc/sudoers.d\n')
     if not os.path.exists('/etc/sudoers.d'):
         os.system('mkdir /etc/sudoers.d && chmod 750 /etc/sudoers.d')
-    if not os.path.exists('/etc/sudoers.d/95-foxpass-sudo'):
-        with open('/etc/sudoers.d/95-foxpass-sudo', 'w') as w:
-            w.write('# Adding Foxpass group to sudoers\n%foxpass-sudo ALL=(ALL:ALL) NOPASSWD:ALL')
-        os.system("chmod 440 /etc/sudoers.d/95-foxpass-sudo")
+    if not os.path.exists('/etc/sudoers.d/95-{sudo}'.format(sudo=sudoers)):
```

Comment:
i think the filename should stay the same

Reply:
could also be bad if user specified a group name with a '/' or some other character that is going to mess up system commands.